packages/indicators/src/builders/buildArithmetic.ts

```diff
@@ -45,8 +49,20 @@ const fetchAnalyticClusters = async (
   const checkClusterIncludesAllElements = (cluster: AnalyticCluster) =>
     allElements.every(member => member in cluster.dataValues);
 
+  const replaceAnalyticValuesWithDefaults = (cluster: AnalyticCluster) => {
+    const returnDataValues = cluster.dataValues;
```

Comment:
Not great practice to mutate a parameter - even by assigning it to this variable, you're still mutating the original (so actually, the `return` becomes redundant)
```suggestion
    const returnDataValues = {...cluster.dataValues};
```

Reply:
😬 Good catch, I didn't notice that